### DISCUSSION TITLE
what is implemented and not?

### DISCUSSION THREAD

**Discussion by vurt72** [exult/exult#295] [General]
Is there a list somewhere of what is implemented in Exult so far and what's not? 

I'm playing BG and i noticed there's no rain though the character comments on it and there's sound of rain. 

Cheers

**Comment by r-h-o-n-a** [comment]
Which version of Exult are you using? I'm running 1.8.0, and for me BG is implementing rain with no problems.

**Reply by vurt72** [reply]
I'm using 1.8, though i am using a higher resolution than the original game, maybe that's the problem.

**Reply by tristantarrant** [reply]
Higher resolution may cause the triggers ("eggs") to not fire reliably

**Reply by vurt72** [reply]
I see, then it was problaby that. Too bad, it looks so good in high resolution. 
is weather scripted with those eggs though? how boring, i've always thought weather is mostly something random. 
This rain started just as i walked out of the gates of the first city.
